src/drv.cc

```diff
@@ -35,7 +36,13 @@ static bool queryIsCached(nix::Store &store,
 
     store.queryMissing(toDerivedPaths(paths), willBuild, willSubstitute,
                        unknown, downloadSize, narSize);
-    return willBuild.empty() && unknown.empty();
+    if (willBuild.empty() && unknown.empty()) {
+        return Drv::CacheStatus::NotBuild;
```

Comment:
I think this conditional is just reversed:

before:

`willBuild.empty() && unknown.empty();` -> isCached == true

after:

`willBuild.empty() && unknown.empty();` -> Cache.NotBuild

Reply:
ok. Could you make a pull request?

Reply:
working on it

Reply:
opened https://github.com/nix-community/nix-eval-jobs/pull/311